config.dist.php

```diff
@@ -1,34 +1,92 @@
 <?php
-// /config-sample.php
+// /config.dist.php
 
 /**
  * The base configuration file for Activiteitenbank
  *
  * You need to copy this file to "config.php" and fill in the values.
- *
- * This file contains the following configurations:
- * - Database settings
- * - Template
- * - baseUrl
- * - absolutePath
  */
 
 return [
+    /**
+     * Settings for connecting to a database.
+     *
+     * @see https://doctrine-orm.readthedocs.org/projects/doctrine-dbal/en/latest/reference/configuration.html#getting-a-connection
+     */
     'dbConnectionParams' => [
-        // fill in connection parameters
-        // (see https://doctrine-orm.readthedocs.org/projects/doctrine-dbal/en/latest/reference/configuration.html#getting-a-connection
-        // for details)
+
     ],
+
+
+    /**
+     * Name of the view template.
+     * The template files need to be located in /view/{template}
+     *
+     * Default value: 'flatly'
+     */
     'template' => 'flatly',
+
+    /**
+     * Domain that the application will run on.
+     * Without trailing slash.
```

Comment:
Maybe specify an example value? What if the application is not running at the root level?